configure.ac

```diff
@@ -483,6 +483,30 @@ AC_CHECK_HEADERS([sys/types.h signal.h stddef.h], [],
 # optional:
 AC_CHECK_HEADERS([sys/time.h locale.h fcntl.h dlfcn.h sys/wait.h sys/sysmacros.h])
 
+# Check for JNI
+AC_ARG_ENABLE([java],
+              [AC_HELP_STRING([--disable-java],
+                              [disable Java Interoperability])])
+AS_IF([test X$enable_java != Xno], [
+   AX_JNI_INCLUDE_DIR
```

Comment:
Right after that you need to traverse `JNI_INCLUDE_DIRS` and maybe append them to `CPPFLAGS`.

Reply:
Fixed